### PR TITLE
doc: add addon folder to store the thinruntime use case

### LINKED ISSUE #2460: [DOC] Add the addon folder to store the thinruntime use cases
**What is the defect and your suggestions on improvement:**
Add the addon folder to store the thinruntime usecase.

### PR DESCRIPTION
**Pull request by wang-mask**
Signed-off-by: wang-mask <2018091609006@std.uestc.edu.cn>

### Ⅰ. Describe what this PR does
Add the addon folder to store the thinruntime usecase and supply the nfs usecase.

### Ⅱ. Does this pull request fix one issue?

fixes #2460 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: addon/nfs/en-nfs.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+First, deploy [NFS Server](https://nfs.sourceforge.net/) on the machine that K8s cluster can access. And configure read and write permissions, and ensure that the NFS service can be accessed on the K8s cluster node.
+
+## Prepare NFS-FUSE Client Image
+1. Parameter Resolution Script
+
+In the FUSE container, you need to extract the configuration information of the remote file system from the relevant **ThinRuntimeProfile, Dataset, and ThinRuntime** resources. The relevant information is saved to the FUSE container in the form of JSON strings in **/etc/fluid/config.json** file.
+
+
+```python
+# fluid_config_init.py
```
Comment (cheyang) [inline addon/nfs/en-nfs.md:13]: 能否分别提供Dockerfile， runtimeProfile.yaml?

Comment (wang-mask) [reply addon/nfs/en-nfs.md:13]: It has in line 82

**Pull request merged**